index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Hello World</title>
+	<link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+	<div class="text">
+		<p>Hello Wold!</p>
+	</div>
+</body>
+</html>
```

Comment:
#### Nitpick :warning:
You don’t have to forget about the use of indentations because it makes your code more understandable.